.github/workflows/pull_request.yml

```diff
@@ -50,8 +50,40 @@ jobs:
         run: pipenv install --dev
       - name: Test translation templates
         run: make test-translation-templates
+  resolve-suggestions-urls:
+    needs: test-translation-templates
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Download translated schemas
+        uses: actions/download-artifact@v1
+        with:
+          name: schemas
+      - name: Install pipenv --dev
+        run: pip install pipenv==2018.11.26
+      - run: |
+          echo "::set-env name=PYTHON_VERSION::$(python -c 'import platform; print(platform.python_version())')"
+      - name: Cache virtualenv
+        id: cache-virtualenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs/
+          key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+      - name: Install virtual environment
+        if: steps.cache-virtualenv.outputs.cache-hit != 'true'
+        run: pipenv install --dev
+      - name: Resolve suggesions urls
+        run: make resolve-suggestions-urls
+      - name: Archive schema artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: schemas
+          path: schemas
```

Comment:
I think we agreed that we'd change the build job to call `build` rather than `build-schemas` so this job wouldn't be necessary?

Reply:
This commit was simply getting pushing back a working script version to allow me to pause for review and respond to comments on other PR's. I haven't yet touched upon the workflow comments.